account_interests/models/res_company_interest.py

```diff
@@ -172,8 +168,14 @@ def create_invoices(self, to_date, groupby='partner_id'):
                 'Creating Interest Invoice (%s of %s) with values:\n%s',
                 idx + 1, total_items, line)
             partner_id = line[groupby][0]
-
             partner = self.env['res.partner'].browse(partner_id)
+
+            # Necesitamos que la factura a generar se cree en un diaro compatible, simulamos crear una nota de debito
+            # para que el odoo auto calcule el diario mas recomendable y usamos ese para crear las factura de interes
+            # relacionada a cada partner
+            journal = self.env['account.move'].with_context(internal_type='debit_note').new(
+                {'res_partner': partner_id, 'move_type': 'out_invoice'}).journal_id
```

Comment:
'res_partner' does not exist in the 'account.move' model, it should be changed to partner_id